components/suggestion/command_provider/command_provider.tsx

```diff
@@ -36,14 +37,19 @@ export class CommandSuggestion extends Suggestion {
             className += ' suggestion--selected';
         }
         let symbolSpan = <span>{'/'}</span>;
-        if (item.IconData === EXECUTE_CURRENT_COMMAND_ITEM_ID) {
+        switch (item.IconData) {
+        case EXECUTE_CURRENT_COMMAND_ITEM_ID:
             symbolSpan = <span className='block mt-1'>{'↵'}</span>;
-        }
-        if (item.IconData === COMMAND_SUGGESTION_ERROR) {
+            break;
+        case COMMAND_SUGGESTION_ERROR:
             symbolSpan = <span>{'!'}</span>;
+            break;
+        case COMMAND_SUGGESTION_CHANNEL:
+            symbolSpan = (<span><i className='icon icon--no-spacing icon-globe'/></span>);
```

Comment:
Do we want to show the "lock" instead of the globe if it's a private channel?

Reply:
The suggestion don't suggest private channels. I guess there is a reasoning for that, and I wonder if we should change that or not.

Reply:
Being consistent with the original channel provider makes sense to me :+1: